### ISSUE TITLE
Feature Request: Open In New Tab

### ISSUE THREAD

**Issue by vanclute** [ebootehsaz/YouTubeNoAds#1]
Great extension, thanks for making it!  The one improvement I can immediately see is to have the option of opening the redirects in a new tab.  This way you don't lose the YT page you were browsing from, that will potentially re-load when you go back to it, losing whatever was there (if it's your home page in particular).

**Comment by ebootehsaz** [comment]
Good point. UBlock Origin starting working again so I didn't see a reason to continue development. If YouTube finds a way to block that, then I'll for sure implement this next.
